src/dataMigration.js

```diff
@@ -43,9 +44,12 @@ export const migrateDataToVersion = async (database, settings) => {
         compareVersions(fromVersion, migration.version) < 0 &&
         compareVersions(toVersion, migration.version) >= 0
       ) {
-        migration.migrate(database, settings);
+        migration.migrate(database, settings, fromVersion, toVersion);
```

Comment:
don't need fromVersion, toVersion in .migrate now

Reply:
3386309da3eeaf60e9290e5d4d3c94e95db44127